test/unit/db/seeds_test.rb

```diff
@@ -5,7 +5,187 @@
+    SimpleLayout.any_instance.expects(:import!).never
+
+
+    Rails.application.load_seed
+
+
+    #### MASTER
+
+    master_account = Account.master
+    assert master_account.approved?
+    assert master_account.state_changed_at.present?
+    assert_equal master_account.id, master_account.provider_account_id
+    assert_equal 'Master Account', master_account.name
+    assert_equal 'master-account', master_account.subdomain
+    assert_equal 'master-account.example.com', master_account.domain
+    assert_equal 'master-account.example.com', master_account.self_domain
+    refute master_account.provider
+    assert_nil master_account.site_access_code.presence
+
+    assert master_account.settings.branding.allowed?
+    assert master_account.settings.end_users.visible?
+    assert master_account.settings.account_plans.visible?
+    assert master_account.settings.service_plans.visible?
+
+    assert_equal 1, master_account.users.count
+    master_user = master_account.users.first!
+    assert_equal 'master', master_user.username
+    assert master_user.minimal_signup?
+    assert master_user.admin?
+    assert master_user.active?
+    assert master_user.activated_at.present?
+
+    assert_equal 1, master_account.services.count
+    master_service = master_account.default_service
+    assert_equal 'Master Service', master_service.name
+    range = [0] | Alert::ALERT_LEVELS
+    assert_equal({ web_buyer: range, email_buyer: range, web_provider: range, email_provider: range}, master_service.notification_settings)
+    assert_equal 'self_managed', master_service.deployment_option
+
+    assert master_service.default_service_plan
+
+    master_app_plan = master_service.default_application_plan
+    assert_equal master_service.id, master_app_plan.issuer_id
+    assert_equal 'enterprise', master_app_plan.name
+
+    assert_equal ApplicationPlan.find_by(name: 'Master Plan').id, master_account.bought_cinstance&.plan.id
+
+    master_account_plan = master_account.default_account_plan
+    assert_equal master_account.id, master_account_plan.issuer_id
+    assert_equal 'Default account plan', master_account_plan.name
+
+    {billing: 'Billing API', account: 'Account Management API', analytics: 'Analytics API'}.each do |system_name, description|
+      assert master_service.metrics.find_by(system_name: system_name, unit: 'hit', friendly_name: description)
+    end
+
+    assert(tenant_app_plan = master_service.application_plans.default)
+    assert_equal 'enterprise', tenant_app_plan.name
+    assert_equal master_service.id, tenant_app_plan.issuer_id
+
+    assert(apicast = master_account.access_tokens.find_by(name: 'APIcast'))
+    assert_equal 'ro', apicast.permission
+    assert_equal %w[account_management], apicast.scopes
+    assert(master_token = master_account.access_tokens.find_by(name: 'Master Token'))
+    assert_equal 'rw', master_token.permission
+    assert_equal %w[account_management], master_token.scopes
+
+    #### TENANT
```

Comment:
WDYT in isolate `test_default_values_for_master` and `test_default_values_for_tenant` ? and maybe create some `assert_default_values` method ? Don't know if it is valid :smile:

Reply:
The problem is it would be very slow to run the seeds more times than necessary 😞 So I dunno what's better... 🤷‍♀

Reply:
makes totally sense.. You can ignore it 😄